include/modem/lte_lc.h

```diff
@@ -722,7 +722,7 @@ struct lte_lc_periodic_search_cfg {
 	 *  when the pattern has ended.
 	 *  If several patterns are configured, this impacts only the last pattern.
 	 */
-	bool loop;
+	uint16_t loop;
```

Comment:
This would have been clearer as a bool. If the type is changed, maybe the used values should be described? Or the type could be changed back to bool and a temp variable used instead?

Reply:
Good point. Changed back to bool.